.github/workflows/pr.yml

```diff
@@ -41,71 +41,75 @@ jobs:
     name: Check for E2E label
     outputs:
       run-e2e: ${{ steps.check-if-pr-has-label.outputs.run-e2e == 'true' && (github.event.action != 'labeled' || (github.event.action == 'labeled' && github.event.label.name == 'ready-for-e2e')) }}
-      run-jobs: ${{ github.event.action != 'labeled' }}
```

Comment:
This boolean condition was not accurate and causing issues running the other jobs